### DISCUSSION TITLE
ToolTip wrap text

### DISCUSSION THREAD

**Discussion by NathanTe** [saadeghi/daisyui#1206] [Q&A] [answered]
When using tooltip in my shopping cart my tooltip text goes off-screen. I have tried using overflow on the div but this only applies to the div, because the tooltip is just placed on top using absolute.

Anyone have any suggestions how to make the text wrap?

![image](https://user-images.githubusercontent.com/61114436/193802274-0d01338a-e064-4d5b-bce7-b316b1e93e16.png)

Thanks in advance

**Comment by saadeghi** [comment, marked as answer]
See this: https://github.com/saadeghi/daisyui/issues/388
